doc/lua_api.txt

```diff
@@ -2495,16 +2495,26 @@ Elements
         * `span=<value>`: number of following columns to affect
           (default: infinite).
 
-### `style[<name 1>,<name 2>,...;<prop1>;<prop2>;...]`
+### `style[<name 1>,<name 2>;<prop1>;<prop2>;...]`
+### `style[<name 1>:<state>,<name 2>:state;<prop1>;<prop2>;...]`
 
 * Set the style for the named element(s) `name`.
+* `state` is a list of states separated by the `+` character.
```

Comment:
```suggestion
* `<state>` is a list of states separated by the `+` character.
```

Reply:
Actually, @SmallJoker specifically asked me to change those to that format so that they better match the format of other formspec elements in the API.